projects/mupdf/project.yaml

```diff
@@ -0,0 +1,4 @@
+homepage: "https://www.mupdf.com"
+primary_contact: "<primary_contact_email>"
```

Comment:
Who is going to be the primary contact? @ccxvii, I guess?

Reply:
@ccxvii is indeed the main author, but if primary_contact is used to report found fuzzing issues it would be even better to refer to our [bugzilla](https://bugs.ghostscript.com/) (report bugs on the Security product if they cause crashes or similar). @ccxvii is not the only one that fixes bugs. :)

Reply:
The contacts need to be email addresses with a google account, so that's not an option unfortunately.

You can put as many email addresses in the "auto_ccs" section as you want, so all the developers who work on security issues can get access :-)

Reply:
> The contacts need to be email addresses with a google account

Does this imply it must be @gmail.com? Or it can be one of those G Suite addresses that are hosted by google but use the company domain?

Reply:
I'm pretty sure GSuite is fine, there are plenty of `auto_ccs` in the repo which aren't @gmail.

Reply:
In most cases I think a G Suite address will work, a Google account that doesn't have a Gmail account would also work (you can sign up with any email address).